[documentation]: Update Iroha Client CLI readme (#3052)
hyperledger/iroha: client_cli/README.md

Check [build and installation instructions](https://hyperledger.github.io/iroha-2-docs/guide/build-and-install.html) for more details.

## Usage 

Run Iroha Client CLI:

```
iroha_client_cli [OPTIONS] <SUBCOMMAND>
```

### Options

|        Option         |                    Description                     |
| --------------------- | -------------------------------------------------- |
| -c, --config <config> | Set a config file path (`config.json` by default). |

### Subcommands

|  Command  |                                                                 Description                                                                 |
| --------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
| `account` | Execute commands related to accounts: register a new one, list all accounts, grant a permission to an account, list all account permissions |
| `asset`   | Execute commands related to assets: register a new one, mint or transfer assets, get info about an asset, list all assets                   |

> | `blocks`  | Get block stream from Iorha peer                                                                                                            |

Reviewer: ```suggestion
| `blocks`  | Get block stream from Iroha peer                                                                                                            |
```

| `domain`  | Execute commands related to domains: register a new one, list all domains                                                                   |
| `events`  | Get event stream from Iroha peer                                                                                                            |
| `json`    | Submit multi-instructions as JSON                                                                                                           |

> | `peer`    | Execute commands related to p2p networking: registering and unregistering peers                                                             |

Reviewer 2: ```suggestion
| `peer`    | Execute commands related to peer administration and networking                                                             |
```

| `wasm`    | Execute commands related to WASM                                                                                                            |

> | `help`    | Print the help message for the tool or a subcommand                                                                                         |

Reviewer 2: ```suggestion
| `help`    | Print the help message for `iroha_client_cli` and/or the current subcommand other than `help` subcommand                                                                                         |
```

These subcommands submit the provided wasm binary as an `Executable` to be executed outside a trigger context.

> ### Execute Multiple Instructions

Reviewer 2: ```suggestion
### Execute Multi-instruction Transactions
```

> You can execute multiple instructions with a single subcommand. Provide a JSON file with multiple instructions related to Iroha Client CLI to execute:

Reviewer 2: ```suggestion
The reference implementation of the Rust client, `iroha_client_cli` is often used for diagnosing problems in other implementations. 

To test transactions in the JSON format (used in the genesis block and by other SDKs), pipe the transaction  into the client and add the `json` subcommand to the arguments: 
```
